docker/cloud/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.20-bullseye AS builder
+WORKDIR /app
+COPY . .
+RUN go build -o nitro .
+
+FROM alpine:latest
```

Comment:
Using build steps like this in docker reduces the image size from ~2.5GB to ~69MB. This works especially well for golang since it is a compiled language and we only need the executable to run the program after its been built so we can discard the source code from the docker image.